pkg/scheduler/ugm/group_tracker.go

```diff
@@ -78,3 +78,7 @@ func (gt *GroupTracker) GetGroupResourceUsageDAOInfo() *dao.GroupResourceUsageDA
 	groupResourceUsage.Queues = gt.queueTracker.getResourceUsageDAOInfo("")
 	return groupResourceUsage
 }
+
+func (gt *GroupTracker) CleanDeletedQueueTracker(queuePath string) {
```

Comment:
Just call it simply `DeleteQueueTracker()`.

Reply:
Good suggestion, addressed in latest PR.